### PR TITLE
Option to match velvet to card color

### PR DESCRIPTION
**Pull request by JackRenoson**
Added an option in the mod config to match the velvet on the mastered cards with the color of the card. This only works on basegame colors, so red, blue, green, purple, curse and colorless. The option is off by default, so that people who do not know about mod config don't have their mod changed without being asked.

### CODE REVIEW

**Comment by erasels** [comment]
Cool contribution, thank you.

Could you please show some screenshots of specifically colorless, curse and purple. Those colors strike me as a bit... odd.
Purple is super saturated (while green is pretty dark), curse is very black and colorless looks piniksh? I think I'd like it more if it wer greyish like the card frame normally would be. 

One issue this has is that it will be incopmpatible with the texture repalcement mod, something I'll have to think about, maybe there's a way to check if it's active.

**Comment by JackRenoson** [comment]
I decided to redo all the art again, using a new tool i found in the image editing software i use. Now they match better with each other. I also madethe curse back a bit lighter, the colorless back a bit darker and the watcher back a bit less saturated. I hope you like these, but if not i'll change them again.
image

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <SlayTheSpire.version>07-05-2019</SlayTheSpire.version>
         <ModTheSpire.version>3.11.0</ModTheSpire.version>
         <!--CHANGE THIS TO YOUR STEAM INSTALLATION-->
-        <Steam.path>E:/SteamLibrary/steamapps</Steam.path>
+        <Steam.path>D:/Games/Steam/steamapps</Steam.path>
```
Comment (erasels) [inline pom.xml]: Please don#t push the pom.xml change :P

**Review by erasels — CHANGES_REQUESTED** [review]
Yeah, I like the redone art much more, thank you.

File: pom.xml (hunk 1 above)
Comment (JackRenoson) [reply pom.xml]: oops, changed it back now

**Comment by erasels** [comment]
Nice, I'll wait with merging this until Kio has a way to check if a texture was replaced by the Texture Replacer mod, so that I can keep compatibility.

I appreciate you taking the time to add to this mod :)

**Comment by JackRenoson** [comment]
Updated to use the new Attack Art and fix the merge issues with the multiple card requirement option
I think this should be fine to be merged. I don't think Kio has added the feature to Texture Replacer, but it still should work, if people have this mod and your other texture replacer mod, it should show the other skin if colored velvet is turned off, and colored velvet for the basegame colors if it is on, and still the other one for modded colors

**Comment by erasels** [comment]
Thank you, the new version with this is out now.

**Pull request merged**